react/.storybook/config.js

```diff
@@ -0,0 +1,34 @@
+import { configure, addDecorator } from '@storybook/react';
+import apolloStorybookDecorator from 'apollo-storybook-react';
+import typeDefs from '../apollo/schema.graphql';
+
+const mocks = {
+  Query: () => {
+    return {
+      status: () => {
+        return "Up"
+      },
+    }
+  },
+  Channel: () => {
```

Comment:
We can obviously get much more interesting/organized about mocks, this is just to get the ball rolling. See: https://www.apollographql.com/docs/graphql-tools/generate-schema.html#example